src/applications/vaos/components/calendar/CalendarNavigation.jsx

```diff
@@ -54,6 +55,7 @@ const CalendarNavigation = ({
           onClick={nextOnClick}
           disabled={nextDisabled}
           type="button"
+          aria-label="Next"
```

Comment:
Axe was flagging the `button` element for the [button-name](https://dequeuniversity.com/rules/axe/4.4/button-name) rule and the aria-label resolves it.

<img width="846" alt="Screenshot 2024-09-25 at 3 28 20 PM" src="https://github.com/user-attachments/assets/e5a90113-8c85-4d9a-a2a2-ef35d63dbe41">
<img width="427" alt="Screenshot 2024-09-25 at 3 28 07 PM" src="https://github.com/user-attachments/assets/1e91ec4a-e7ed-46ae-8572-8c62636d9818">